<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="Przelaczalny_widzet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="Macierz_odleglosci_3">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <attribute name="title">
     <string>Macierz odległości</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="DistanceLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="distance_matrix_load_3">
        <property name="text">
         <string>Wczytaj...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="distance_matrix_save_3">
        <property name="text">
         <string>Zapisz zmiany</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Macierz odległości pomiędzy sklepami</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="EditDistanceMatrix_3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Macierz_cen_3">
    <attribute name="title">
     <string>Macierz cen</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="PriceLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="price_matrix_save_3">
        <property name="text">
         <string>Zapisz zmiany</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="EditPriceMatrix_3"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="price_matrix_load_3">
        <property name="text">
         <string>Wczytaj...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Macierz cen produktów w sklepach</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Wektor_poczatkowy_3">
    <attribute name="title">
     <string>Wektor początkowy</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>651</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="VectorLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="primary_vector_save_3">
        <property name="text">
         <string>Zapisz zmiany</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="EditPrimaryVector_3"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="primary_vector_load_3">
        <property name="text">
         <string>Wczytaj...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Wektor początkowy(kolumna 1: numer sklepu, kolumna 2: numer produktu)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Wynik_3">
    <attribute name="title">
     <string>Wynik</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Licz Rozwiązanie</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>211</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ilość sklepów</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ilość produktów</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Długość listy tabu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ilość iteracji</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Plik wyjścowy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Macierz odległości</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Macierz cen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Wektor początkowy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_sklepy"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_produkty"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_tabu"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_iteracje"/>
        </item>
        <item>
         <widget class="QLineEdit" name="nazwa_pliku"/>
        </item>
        <item>
         <widget class="QLineEdit" name="matrix_name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="price_name"/>
        </item>
        <item>
         <widget class="QLineEdit" name="vector_name"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="warning">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>130</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Podglad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Podgląd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Najlepsze rozwiązanie</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Optymalna wartośc funkcji celu:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FCValue">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="wektorOptymalny">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>191</width>
       <height>241</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="WykresTab_3">
    <attribute name="title">
     <string>Wykres</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>891</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="plotLayout_3">
      <item row="0" column="0">
       <widget class="QCustomPlot" name="customPlot_3" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Generuj macierz</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>291</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Wiersze</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Kolumny</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Plik wyjściowy</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Min</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Max</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Zablokowana przekątna</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLineEdit" name="lineEditWiersze"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditKolumny"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPlik"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMax"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxPrzekatna">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generuj!</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="czyWektor">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Wektor</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
